app/src/main/res/layout/drag_drop_single_item.xml

```diff
@@ -10,14 +10,10 @@
 
   <TextView
     android:id="@+id/drag_drop_content_text_view"
-    android:layout_width="wrap_content"
-    android:layout_height="wrap_content"
-    android:fontFamily="sans-serif"
+    style="@style/Subtitle1"
     tools:text="Hello"
```

Comment:
This tools text shouldn't be here, can you please remove it? I know you didn't add it originally, but this is a good opportunistic fix. :)

Reply:
Done